<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">MControlCenter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="infoTab">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="title">
        <string>Info</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="ecVersionLabel">
          <property name="text">
           <string>EC Version:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="ecVersionValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ecBuildLabel">
          <property name="text">
           <string>EC Build:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="ecBuildValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="BatteryChargeLabel">
          <property name="text">
           <string>Battery charge:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="BatteryChargeValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="batteryThresholdLabel">
          <property name="text">
           <string>Battery threshold:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="batteryThresholdValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="chargingStatusLabel">
          <property name="text">
           <string>Charging status:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="chargingStatusValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="cpuTempLabel">
          <property name="text">
           <string>CPU temp:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="cpuTempValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="gpuTempLabel">
          <property name="text">
           <string>GPU temp:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="gpuTempValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Fan 1:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="fan1ValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Fan 2:</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="fan2ValueLabel">
          <property name="text">
           <string notr="true">-</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QCheckBox" name="coolerBoostCheckBox">
          <property name="text">
           <string>Cooler Boost</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="modeTab">
       <attribute name="title">
        <string>Mode</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="highPerformanceModeRadioButton">
            <property name="text">
             <string>High Performance</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="balancedModeRadioButton">
            <property name="text">
             <string>Balanced</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>A mode that brings a balance between performance and battery live</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="silentModeRadioButton">
            <property name="text">
             <string>Silent</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="superBatteryModeRadioButton">
            <property name="text">
             <string>Super Battery</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>A mode that provides the best system performance for multi-tasking and heavy duty</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>A quiet mode that lowers the fan noise</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>A power-saving mode that lowers the power consumption for basic needs</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="batteryTab">
       <attribute name="title">
        <string>Battery</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>If you mainly use your laptop with adaptor plugged, we suggest you set the charge capacity at a lower percentage (60% or 80%) to prolong your battery lifecycle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="horizontalSpacing">
           <number>10</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="bestMobilityRadioButton">
            <property name="text">
             <string>Best for Mobility</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Charge the battery to 100% all the time</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="balancedBatteryRadioButton">
            <property name="text">
             <string>Balanced</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Charge the battery when under 70%, stop at 80%</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="bestBatteryRadioButton">
            <property name="text">
             <string>Best for battery</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Charge the battery when under 50%, stop at 60%</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QRadioButton" name="customBatteryThresholdRadioButton">
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="customBatteryThresholdComboBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string notr="true">20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">35</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">40</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">45</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">55</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">60</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">65</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">70</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">75</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">80</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">85</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">90</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">95</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">100</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="keyboardBacklitLabel">
          <property name="text">
           <string>Keyboard Backlit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="keyboardBacklightModeComboBox">
          <item>
           <property name="text">
            <string>Always on</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auto turn off in 10 sec</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="keyboardBrightnessSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximum">
           <number>3</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="fnSuperSwapCheckBox">
          <property name="toolTip">
           <string>Swap FN and Super buttons</string>
          </property>
          <property name="text">
           <string>FN ⇄ Super</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="webCamCheckBox">
          <property name="text">
           <string>WebCam</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="usbPowerShareCheckBox">
          <property name="text">
           <string>USB Power Share</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="debugTab">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <widget class="QSpinBox" name="ValueSpinBox">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="addressEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>60</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>54</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true">Address</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>30</y>
          <width>54</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true">Value</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ReadValueButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true">Read Value</string>
        </property>
       </widget>
       <widget class="QPushButton" name="WriteValueButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true">Write Value</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="aboutTab">
       <attribute name="title">
        <string>About</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="1">
         <layout class="QFormLayout" name="aboutTabFormLayout">
          <property name="rowWrapPolicy">
           <enum>QFormLayout::WrapLongRows</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="versionLabel">
            <property name="text">
             <string>Version:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="gitHubLabel">
            <property name="text">
             <string>GitHub:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="gitHubValueLabel">
            <property name="text">
             <string notr="true">&lt;a href=&quot;https://github.com/dmitry-s93/MControlCenter&quot;&gt;https://github.com/dmitry-s93/MControlCenter&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextBrowserInteraction</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="bugTrackerLabel">
            <property name="text">
             <string>Bug tracker:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="bugTrackerValueLabel">
            <property name="text">
             <string notr="true">&lt;a href=&quot;https://github.com/dmitry-s93/MControlCenter/issues&quot;&gt;https://github.com/dmitry-s93/MControlCenter/issues&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::TextBrowserInteraction</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="versionValueLabel">
            <property name="text">
             <string notr="true">0.2.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>An application that allows you to change the settings of MSI laptops running Linux</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
